src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAOStub.java

```diff
@@ -41,7 +42,7 @@ public MediaEntry save(MediaEntry mediaEntry) throws Exception {
      * @return List of MediaEntry objects belonging to the given user
      */
     @Override
-    public List<MediaEntry> fetchByUsername(String username) throws Exception {
+    public List<MediaEntry> fetchByUsername(String username) {
```

Comment:
There were a few places that throws exception wasn't being used so i just removed it to clean up your code.
https://www.baeldung.com/java-clean-code